db/migrate/20220114145558_migrate_was_employed_in_claims.rb

```diff
@@ -0,0 +1,15 @@
+class MigrateWasEmployedInClaims < ActiveRecord::Migration[6.1]
+  class Claims < ActiveRecord::Base
+    self.table_name = :claims
+  end
+
+  def up
+    Claim.all.each do |was_employed|
```

Comment:
There are a couple of issues here

1. You are looping over all claims and calling the item within the loop 'was_employed'.  This is very confusing when it is a claim ?
2. Once I replace the first 'was_employed' with 'claim' in my mind, you are doing 
```ruby

claim.update_columns was_employed: claim.was_employed.present?
```
This is definitely not right - go back to the original definition and work out what it should be